frontend/packages/kubevirt-plugin/integration-tests/tests/models/vmwareImportWizard.ts

```diff
@@ -132,13 +132,14 @@ export class VmwareImportWizard extends ImportWizard {
     await this.next();
 
     // Advanced - Virtual HW
-    await this.next();
+    // await this.next();
```

Comment:
this is gone now. Please remove it properly

Reply:
Commented line removed